Jenkinsfile

```diff
@@ -151,6 +151,7 @@ pipeline {
               dir("${BASE_DIR}"){
                 sh script: './scripts/jenkins/bench.sh', label: 'Benchmarking'
                 sendBenchmarks(file: 'build/bench.out', index: 'benchmark-go')
+                generateGoBenchmarkDiff(file: 'build/bench.out', filter: 'exclude')
```

Comment:
I'm not sure if `file` should be a filename or a relative path with nested folders...